test/test_nestedtensor.py

```diff
@@ -230,7 +230,7 @@ def layout_name(layout):
 
 def get_op_name(layout):
     # e.g. "<OpOverload(op='aten.sum', overload='dim_IntList')>" -> "sum"
-    return layout.__name__.split(".")[0]
+    return layout.__name__.split(".")[0].split("_")[-1]
```

Comment:
curious why you need the `_` ?

Reply:
depending on the op, `layout.__name__.split(".")[0]` returns `_softmax` rather than `softmax`, so wanted to remove the `_`!